tools/cvm-image-rewriter/plugins/60-initrd-update/README.md

```diff
@@ -0,0 +1,3 @@
+# Initrd update
+
+This plugins is used to update the initrd image. Users place executable files in the `files/` directory, and the plugin will copy these executable files to the initramfs.
```

Comment:
Is an initrd considered an executable file?   Would it be better to say  something like  "To change the initrd image, place the necessary files in the `files/` directory. The plugin will copy these files to the initramfs."